### ISSUE TITLE
Problem deploying Komadu as a Web Service

### ISSUE THREAD

**Issue by OYang007** [Data-to-Insight-Center/komadu#14]
Hi,
  I am a student studying provenance database.
  I want to collection provenance information from workflow by komadu.
  I was trying to get the Komadu runninig ,following the User Guide provided.

  (1) What I'd like to know is which of the two methods of running Komadu, web service or standalone server, would be suitable.
  (2) After executing command “mvn clean install -Dmaven.test.skip = true”, i got the file “komadu-service-1.0.0-SNAPSHOT.aar”,not “komadu-service-1.0.aar”, so i failed deploy Komadu as a Web Service.
Example Errors:
[ERROR] The komadu-service-1.0.0-SNAPSHOT.aar service, which is not valid, caused A ClassNotFoundException error occurred in loading the message receiver edu.indiana.d2i.komadu.service.KomaduServiceMessageReceiverInOnly org.apache.axis2.deployment.DeploymentException: A ClassNotFoundException error occurred in loading the message receiver edu.indiana.d2i.komadu.service.KomaduServiceMessageReceiverInOnly

System Details:
Window 10 professional , maven 3.3.9, tomcat 7.072, axis2-1.72, mysql-5.6.29

 That is above all,could you please give me some suggestions ?
Sincerely,
OYang007

**Comment by isurues** [comment]
Hi,

If you want to use komadu 1.0 released version, please download the 1.0 source from the following release location.

https://github.com/Data-to-Insight-Center/komadu/releases

And also, please follow the user guide found inside the docs directory in the downloaded zip archive. If you still run into issues, please let us know.

Thanks,
Isuru

**Comment by OYang007** [comment]
Hi,@isurues
Thanks for your reply, but i failed to deploy Koamdu as a Web Service again.

I used default value except for log4j.properties.path, database.username,database.password.
I downloaded commons-lang-2.6.jar from http://mvnrepository.com/artifact/commons-lang/commons-lang/2.6 because i did not find ~m2\repository\commons-lang\commons-lang\2.6 directory.
I builded prov-xml under the ProvToolbox-ProvToolbox-0.4.0 directory because of failing to build it under prov-xml directory.

[ERROR] The komadu-service-1.0.0.aar service, which is not valid, caused A ClassNotFoundException error occurred in loading the message receiver edu.indiana.d2i.komadu.service.KomaduServiceMessageReceiverInOnly

Thanks,
OYang
